.github/workflows/typescript.yml

```diff
@@ -16,8 +16,8 @@ jobs:
         node-version: '18.x'
     - uses: actions/checkout@v4
     - name: Build
-      working-directory: ./docs/servo_power_monitor/
+      working-directory: ./docs/servo_power_test/
       run: npm install
     - name: Run tests
-      working-directory: ./docs/servo_power_monitor/
+      working-directory: ./docs/servo_power_test/
```

Comment:
I think we have to duplicate the github actions workflow for both of the tools (servo_power_monitor, servo_power_test). Please update the YAML file to do tests on both of them 🙏